src/style.css

```diff
@@ -0,0 +1,82 @@
+@font-face {
+  font-family: Avenir-Book;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+html {
+  font-weight: 300;
+}
+
+.search-form {
+  margin-top: 20px;
+}
+
+.search {
+  width: 100%;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  font-family: Avenir, sans-serif;
+  margin-top: 20px;
```

Comment:
Check this margin, I think it messes the look (it is a good practice to alway run spn start after you make some changes to see if it is not destroying anything) :) :
<img width="374" alt="image" src="https://user-images.githubusercontent.com/126275137/236662415-77de92ee-d030-4f90-b06e-182646600a9d.png">